site/gatsby-site/src/components/submissions/SubmissionForm.js

```diff
@@ -110,17 +113,9 @@ const SubmissionForm = () => {
 
   const [parsingNews, setParsingNews] = useState(false);
 
-  const coldStartToast = () => {
-    addToast({
```

Comment:
Cold starts are no longer a problem then?

Reply:
Yup

Reply:
I mean, it when from the 20 to 30 secs to only a few.